### PR TITLE
chore: Comply with AIMs Web Add/Drop Page update

### PR DESCRIPTION
**Pull request by TheephopWS**
CityU have changed the UI of the Course Adding page and the link need to be undated correspondingly 🙏

### CODE REVIEW

File: manifest.json
Comment (tLok922) [inline manifest.json:1]: Suggestion - "matches": ["https://banappstureg.cityu.edu.hk/*", "http://banappstureg.cityu.edu.hk/*"] to include http, but it seems that the new website doesn't connect when http is used. 

You can 1) remove duplicates, 2) change one URL to http

Comment (H11Maitree) [reply manifest.json:1]: I suggest to 
3) **Stay to use two URL both `https` AND `http`** (keep the original logic)
just incase CityU change the config or `https` certificate down lol

ps. Personally, I think this is unintentional code and this is just his typo.
context: The having both `http` and https is not my decision it the original codebase from Sam (CityU grads who initiate this code)

@Aobcoffee Can you edit on this pls ✍️

**Comment by H11Maitree** [comment]
I [re-added](https://github.com/H11Maitree/CityU-Course-Adder-Chorme-Extension/pull/1/commits/e26b4e07bd476c0b403ab0027ff1f6e40029278e) the original `contentScript` at `legacy/contentScript.js` into our codebase to ensure **backward compatibility with the old CityU Course Adding Page**. This ensures that if CityU ever revert back to the older system, our functionality remains intact.

@Aobcoffee Pls keep this [backward compatibility awareness] in to your consideration for your future PR in any repo 🫡.

**Comment by TheephopWS** [comment]
I did update the typo. Thank you for your consideration 🙏

**Review by H11Maitree — APPROVED** [review]

**Pull request merged**